frontend/src/providers/graphqlProvider.tsx

```diff
@@ -1,7 +1,7 @@
 import { ApolloClient, InMemoryCache, ApolloProvider } from "@apollo/client";
 
 const client = new ApolloClient({
-    uri: "http://localhost:8080/graphql",
+    uri: process.env.NEXT_PUBLIC_GRAPHQL_ENDPOINT || "http://localhost:8080/graphql",
```

Comment:
I prefer to have a .env file to define the variables values instead of defining default values in the source code.

Reply:
I understand that these `NEXT_PUBLIC_` variables should be defined at build time, and will make part of the client side code.

Even if using a `.env`, we could load the `.env` before calling the `build`, right?

Reply:
I think I agree with @claudioantonio. Maybe we should have an `.env` file for debugging, which makes heavy use of `localhost`, and another (that might not be committed) for production.

Reply:
@endersonmaia I opened a draft PR for the env vars. Please check #56 and let me know if it helps you.